src/components/inputs/TextField.js

```diff
@@ -217,21 +217,21 @@ export default class TextField extends BaseInput {
   }
 
   getAccessibilityInfo() {
-    const {floatingPlaceholder, placeholder, expandable, value} = this.getThemeProps();
+    const {floatingPlaceholder, placeholder, expandable} = this.getThemeProps();
+    const accessibilityState = this.isDisabled() ? {disabled: true} : undefined;
+    let accessibilityLabel = floatingPlaceholder ? this.props.accessibilityLabel || placeholder : '';
 
-    let accessibilityLabel = floatingPlaceholder ? placeholder : undefined;
     if (this.isRequiredField()) {
-      accessibilityLabel = `${accessibilityLabel || ''}. Mandatory`;
+      accessibilityLabel = `${accessibilityLabel}. Mandatory`;
     }
     if (expandable) {
-      accessibilityLabel = `${accessibilityLabel || ''}. ${value || ''}`;
+      accessibilityLabel = `${accessibilityLabel}`;
     }
 
-    const accessibilityStates = this.isDisabled() ? ['disabled'] : [];
     return {
       accessibilityLabel,
       // on Android accessibilityStates cause issues with expandable input
-      accessibilityStates: Constants.isIOS ? accessibilityStates : undefined
+      accessibilityState: Constants.isIOS ? accessibilityState : undefined
```

Comment:
I think `accessibilityState` (singular) is relevant to the new RN version (rn62)
I think in rn61 they use `accessibilityStates` (plural) 
All other places we use in our code are plural. 
Did u check it works? 
Also why are u changing it.. I believe we have just fixed it recently to
`const accessibilityStates = this.isDisabled() ? ['disabled'] : [];`
on purpose, it solved a bug.

Reply:
this: `const accessibilityStates = this.isDisabled() ? ['disabled'] : [];` doesn't work now...
Why would I change it if it doesn't work 🤷‍♂.

Reply:
ok